dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/zk/ZKMasterClient.java

```diff
@@ -16,6 +16,9 @@
  */
 package org.apache.dolphinscheduler.server.zk;
 
+import java.util.Date;
```

Comment:
Looks like these files not need to be changed, you should revert these changes.
The same with `TaskResponseService.java`, `TaskResponseEvent.java`, `TaskExecuteResponseCommand.java`, `BaseDataSource.java `, `common.properties`.